test/azure/RspecTests/models_generation_and_inheritance_spec.rb

```diff
@@ -35,6 +35,7 @@
     # Should generate Resource class
     resource = modules.const_get('Resource')
     expect(resource.is_a?(Class)).to be_truthy
+    expect(resource.new.methods.include? :resource_group).to be_truthy
```

Comment:
tests are on azure generator size, so code should be there as well

Reply:
are there any negative cases where we should check that resource_group did not get included?

Reply:
Done